src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestDfpParamsPresentMobile.java

```diff
@@ -20,7 +20,7 @@
       dataProvider = "dfpParams",
       groups = {"MobileAds", "TestDfpParamsPresentMobile_GeoEdgeFree"}
   )
-  public void TestDfpParamsPresentMobile_GeoEdgeFree(
+  public void testDfpParamsPresentMobile_GeoEdgeFree(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)